log.go

```diff
@@ -161,24 +160,31 @@ func (l Level) String() string {
 // ParseLevel converts a level string into a zerolog Level value.
 // returns an error if the input string does not match known values.
 func ParseLevel(levelStr string) (Level, error) {
-	switch strings.ToLower(levelStr) {
-	case LevelFieldMarshalFunc(TraceLevel):
+	if strings.EqualFold(levelStr, LevelFieldMarshalFunc(TraceLevel)) {
 		return TraceLevel, nil
-	case LevelFieldMarshalFunc(DebugLevel):
+	}
+	if strings.EqualFold(levelStr, LevelFieldMarshalFunc(DebugLevel)) {
```

Comment:
please keey using a switch/case with the condition.

Reply:
You mean I should revert the last commit?
Because as I explained before the `EqualFold` does not work with `switch` statement.

Reply:
Use the EqualFold as the expression of the switch with an empty switch.

Reply:
Alright, the expressions were exchanged.